sdgx/data_models/metadata.py

```diff
@@ -44,6 +44,16 @@ class Metadata(BaseModel):
     column_list is used to store all columns' name
     """
 
+    column_inspect_level: Dict[str, int] = defaultdict()
```

Comment:
Backwards compatibility does not exist for `inspect_level` as `10`.

```suggestion
    column_inspect_level: Dict[str, int] = defaultdict(lambda: 10)
```